src/components/GetMakeCodeCodeViewSvg.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button } from "@chakra-ui/react";
+import {
+  BlockLayout,
+  MakeCodeProject,
+  useMakeCodeRenderBlocks,
+} from "@microbit-foundation/react-code-view";
+import { useCallback } from "react";
+
+// Non-user-facing component for development purposes
+// to log SVG of MakeCode block
+
+const GetMakeCodeCodeViewSvgButton = ({
+  code: project,
+}: {
+  code: MakeCodeProject;
+}) => {
+  const { renderBlocks } = useMakeCodeRenderBlocks({});
+  const handleOnClick = useCallback(async () => {
+    const result = await renderBlocks({
+      code: project,
+      options: { layout: BlockLayout.Flow },
+    });
+    console.log(result.svg);
```

Comment:
Remove debug.
```suggestion
```

Reply:
This component is not currently used and is just for getting the svg for rendering the default blocks. The `console.log` is for printing out the svg for development purposes, not sure if I should still remove the `console.log` in this case.